clients/cobol-lsp-vscode-extension/src/services/JavaCheck.ts

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright (c) 2020 Broadcom.
+ * The term "Broadcom" refers to Broadcom Inc. and/or its subsidiaries.
+ *
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Broadcom, Inc. - initial API and implementation
+ */
+import * as cp from "child_process";
+
+export class JavaCheck {
+    public static isJavaVersionSupported(versionString: string) {
+        // tslint:disable-next-line: max-line-length
+        const versionPattern = new RegExp('(java|openjdk) (version)? ?"?((9|[0-9][0-9])|(1|9|[0-9][0-9])\.(1|8|[0-9][0-9]).*).*');
```

Comment:
The new Java to be released next week, we should if this regex is actual later on